param/coercer/coercer_test.go

```diff
@@ -27,10 +28,75 @@ func TestCoerceParams_IntegerCoercion(t *testing.T) {
 		"intkey": "123",
 	}
 
-	CoerceParams(schema, data)
+	err := CoerceParams(schema, data)
+	assert.NoError(t, err)
 	assert.Equal(t, 123, data["intkey"])
 }
 
+func TestCoerceParams_IntegerIndexedMapCoercion(t *testing.T) {
+	{
+		schema := &spec.JSONSchema{Properties: map[string]*spec.JSONSchema{
+			"arraykey": {Type: []string{arrayType}},
+		}}
+		data := map[string]interface{}{
+			"arraykey": map[string]interface{}{
+				"0": "0-index",
+				"2": "2-index",
+			},
+		}
+
+		err := CoerceParams(schema, data)
+		assert.NoError(t, err)
+		assert.Equal(t, "0-index", data["arraykey"].([]interface{})[0])
```

Comment:
Would be nice to also check `2-index`

Reply:
Cool! Added that and also another test that the 1st element is a `nil` as expected.